### PR TITLE
Show more informations about functions in the external-solution test

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Expand one more information when a function is deploying in external-solution test

### CODE REVIEW

File: tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_function.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func (s *DeployFunction) isFunctionReady() error {
 	}
 
 	if !s.isReady(function) {
-		return errors.New("function is not ready yet")
+		return fmt.Errorf("function \" %+v \"is not ready yet", function)
```
Comment (dbadura) [inline tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_function.go:160]:
```suggestion
		return fmt.Errorf(`function "%+v" is not ready yet`, function)
```

Comment (magicmatatjahu) [inline tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_function.go:160]: Is quotes necessary? Maybe use new lines.

Comment (pPrecel) [reply tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_function.go]: Good point

**Pull request merged**